README.md

```diff
@@ -1,39 +1,37 @@
-# FHIR Information Gateway
+# FHIR Info Gateway
 
 <!-- Build status of the main branch -->
 
 [![Build Status](https://storage.googleapis.com/fhir-proxy-build-badges/build.svg)](https://storage.googleapis.com/fhir-proxy-build-badges/build.html)
 
-FHIR Information Gateway is a simple access-control proxy that sits in front of
-a [FHIR](https://www.hl7.org/fhir/) store (e.g., a
+The FHIR Info Gateway is an access-control proxy that sits in front of a
+[FHIR](https://www.hl7.org/fhir/) store (e.g., a
 [HAPI FHIR](https://hapifhir.io/) server,
 [GCP FHIR store](https://cloud.google.com/healthcare-api/docs/concepts/fhir),
 etc.) and controls access to FHIR resources.
 
-Note: "gateway" and "proxy" are used interchangably here, as the gateway is
-implemented as a proxy server.
```

Comment:
Please keep this; there are still references to "proxy" in this doc and I don't want all of them to be replaced with "FHIR Info Gateway".